tests/unit_tests/scheduler/test_scheduler.py

```diff
@@ -546,6 +546,15 @@ def test_scheduler_create_lsf_driver():
     assert driver._resource_requirement == lsf_resource
 
 
+def test_scheduler_create_lsf_driver_no_excluded_hosts():
+    queue_config_dict = {
+        "QUEUE_SYSTEM": "LSF",
+    }
+    queue_config = QueueConfig.from_dict(queue_config_dict)
+    driver: LsfDriver = create_driver(queue_config)
+    assert len(driver._exclude_hosts) == 0
```

Comment:
I will claim that you are here testing an implemementation detail of the driver, there is little guarantee that this test will catch a future bug. There are some tests utilizing the `capturing_bsub` fixture, I think that might be better to use, and assert that `hname` is not part of the string sent to bsub.

Reply:
Yes, I updated one of the `capturing_bsub` tests to incloude `assert 'hname' not in <QSUB_ARGS>`, and the test passes after this commit but not prior.👌